routing/turns.hpp

```diff
@@ -16,17 +16,19 @@ namespace routing
 /// and a direction.
 struct SegmentRange
 {
+  friend string DebugPrint(SegmentRange const & segmentRange);
```

Comment:
std::string

Reply:
Not now. Pls have a look at the top of the file. `#include "std/string.hpp"`

Reply:
Done